mitodl/open-discussions
# Remove Dockerfile line related to moira and ssl

#### Pre-Flight checklist



#### What are the relevant tickets?
Related to #3767 

#### What's this PR do?
- Removes moira-related line hack from dockerfile which has no effect on heroku

#### How should this be manually tested?
Run `docker-compose build`

Copy settings for `MIT_WS_CERTIFICATE` and `MIT_WS_PRIVATE_KEY` from heroku RC into your .env, but as one-liners (I can send you the formatted values via keybase).

Run the following, you should not get an ssl exception:
```python
from moira_lists.moira_api import *
get_moira_client()
```
